### PR TITLE
Remove storybook

### PR DESCRIPTION
**Pull request by firattale**
Removing storybook from everywhere

### CODE REVIEW

**Review by vidvidvid — COMMENTED** [review]
`yarn.lock` changes are missing - did you run `yarn` after removing the packages?
Also the deployment is failing for some reason - @dysbulic could you take a look at what's happening, please?

**Review by vidvidvid — CHANGES_REQUESTED** [review]
A couple of more packages are left to uninstall - please double check they are not used anywhere else and throw them into the abyss.

File: packages/design-system/.storybook/main.js [hunk 1]
```diff
@@ -1,47 +0,0 @@
...
-    '@storybook/addon-links',
-    '@storybook/addon-docs',
-    'storybook-addon-react-docgen',
-  ],
-  webpackFinal: async (config) => {
-    config.module.rules.push({
-      test: /\.(ts|tsx)$/,
-      use: [
-        {
-          loader: require.resolve('ts-loader'),
```
[the remark this answers is not in the document]
Comment (firattale) [reply packages/design-system/.storybook/main.js:17]: I removed them, thanks for noticing.

[the remark this answers is not in the document]
Comment (vidvidvid) [reply packages/design-system/.storybook/main.js:17]: np, thanks for removing

**Review by vidvidvid — APPROVED** [review]

**Comment by vidvidvid** [comment]
We can merge!

**Pull request merged**